content/ri/using-redisinsight/secure-database.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: Secure Redis database
```

Comment:
Maybe "Secure work with databases"?
do not know if "Secure database" sends the right message, cause we just ask for username and password.